### PR TITLE
66 create a may produce false information message at the bottom of chats

### PR DESCRIPTION
**Pull request by ritikk7**
warning message added

### CODE REVIEW

**Review by kylexrich — CHANGES_REQUESTED** [review]
Looks great! Do you think that we might be producing false information about people, places, and facts though? Or are there other categories that might be more applicable to CourseGPT?

**Comment by ritikk7** [comment]
> Looks great! Do you think that we might be producing false information about people, places, and facts though? Or are there other categories that might be more applicable to CourseGPT?

I consulted ChatGPT on this lol. People, Places, and Facts sum up most of what an AI could produce wrong in a single short line. In our case People could mean outdated instructor info, places could mean outdated class/exam locations, and well facts could mean any course content so I didn't bother changing that.

File: client/src/components/organisms/RightSection/RightSection.js [hunk 1]
```diff
@@ -42,6 +48,7 @@ const InputArea = ({
         <ArrowForwardIcon />
       </button>
     </div>
+    <Message value={"CourseGPT may produce inaccurate information about people, places, or facts. [CourseGPT June 30 Version]"}/>
```
Comment (kylexrich) [inline client/src/components/organisms/RightSection/RightSection.js:51]: What about: "CourseGPT may produce inaccurate information about instructors or course information. [CourseGPT 2023 Version]"

**Review by kylexrich — CHANGES_REQUESTED** [review]

**Comment by huangcar1** [comment]
I think that both make sense + it depends on if we want to go for something more general/specific! I think I prefer the specific one, but I don't really have a strong preference. The date should definitely be the year, though, otherwise we'll have to change it constantly or it'll look outdated.

**Comment by ajo01** [comment]
I'm fine with either. The specific version is exactly what ChatGPT has right now. Also agree with only putting up the year.

**Comment by duffydu** [comment]
I would also prefer the more specific version. And +1 for the year

**Review by ajo01 — APPROVED** [review]

**Review by duffydu — APPROVED** [review]

**Review by huangcar1 — APPROVED** [review]

**Review by kylexrich — APPROVED** [review]

**Pull request merged**